### PR TITLE
Correctly handle the end time being 0 in the URL.

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: web/ui/static/js/prom_console.js [hunk 1]
```diff
@@ -138,10 +138,17 @@ PromConsole.TimeControl.prototype._setHash = function() {
...
 PromConsole.TimeControl._initialValues = function() {
   var hash = window.location.hash;
+  var values;
   if (hash.indexOf('#pctc') === 0) {
-    return JSON.parse(decodeURIComponent(hash.substring(5)));
+    values = JSON.parse(decodeURIComponent(hash.substring(5)));
+  } else {
+    values = {duration: 3600, endTime: 0};
+  }
+  if (values.endTime == 0) {
```
Comment (juliusv) [inline web/ui/static/js/prom_console.js]: `===` would be stricter.

**Comment by juliusv** [comment]
:+1:

File: web/ui/static/js/prom_console.js (hunk 1 above)
Comment (brian-brazil) [inline web/ui/static/js/prom_console.js]: This way will catch the empty string too.

Comment (juliusv) [inline web/ui/static/js/prom_console.js]: Ugh. Yay for implicitness... it'll also catch `[]` and who knows what in JS too :)

**Pull request merged**